app/models/freak.rb

```diff
@@ -5,6 +5,7 @@ class Freak < ApplicationRecord
 
   has_many :freaks_projects, dependent: nil, class_name: 'FreakProject'
   has_many :projects, through: :freaks_projects, dependent: nil
```

Comment:
We can probably use https://guides.rubyonrails.org/association_basics.html#the-has-and-belongs-to-many-association instead of these. Same for the technologies association.

Reply:
Why we need to change has_many through into has_and_belongs_to_many?

Reply:
Because it's the same thing, but shorter & cleaner.